bom/pom.xml

```diff
@@ -45,27 +45,26 @@
       https://bugs.openjdk.java.net/browse/JDK-8230305
      -->
     <java-base-image.name>docker.io/eclipse-temurin:17-jre-focal</java-base-image.name>
-    <javax.annotation.api.version>1.3.2</javax.annotation.api.version>
     <jjwt.version>0.11.2</jjwt.version>
+    <!--
+      Explicitly override Kafka 3.0 client version defined by Quarkus because
+      vertx-kafka-client 4.2.x is incompatible with Kafka 3.0 client.
+      See https://github.com/quarkusio/quarkus/issues/22000
+    -->
+    <kafka-client.version>2.8.1</kafka-client.version>
     <kafka.image.name>confluentinc/cp-kafka:6.2.2</kafka.image.name>
-    <k8s-client.version>5.10.2</k8s-client.version>
-    <log4j-api.version>2.17.1</log4j-api.version>
     <logback.version>1.2.9</logback.version>
-    <micrometer.version>1.7.6</micrometer.version>
     <mongodb-image.name>mongo:4.4</mongodb-image.name>
-    <mongodb-driver.version>4.3.4</mongodb-driver.version>
     <native.image.name>registry.access.redhat.com/ubi8/ubi-minimal:8.4</native.image.name>
     <netty.version>4.1.72.Final</netty.version>
     <netty.tcnative.version>2.0.46.Final</netty.tcnative.version>
-    <opentracing.version>0.33.0</opentracing.version>
```

Comment:
@sophokles73 Only noticed this now:
The `opentracing.version` property is still referenced in the hono-core pom in the `maven-bundle-plugin` definition.
Without the property defined, a Hono build with the `embed-dependencies` maven profile fails.

Reply:
I guess we (that is: you) can create an issue for it. However, I doubt that anybody is (still) trying to deploy the client module as an OSGi bundle.